google-cloud-tools-plugin/testData/project/custom.yaml

```diff
@@ -0,0 +1,25 @@
+runtime: custom
+service: customService
+api_version: 1
+threadsafe: true
+
+handlers:
```

Comment:
don't include any of the below stuff in tests for now.  most of it is unsupported in app engine flex